### ISSUE TITLE
Support implicit anchors

### ISSUE THREAD

**Issue by jgerigmeyer** [oddbird/css-anchor-positioning#273]
Once https://github.com/whatwg/html/pull/9144 is resolved, we should add support for implicit anchors and the `position-anchor: auto` keyword.

https://drafts.csswg.org/css-anchor-position/#implicit

**Comment by jamesnw** [comment]
You're correct that the anchor attribute is not going to happen. Implicit anchors still are possible, primarily with popovers (including customizable select) and pseudo elements.

I'm assuming you are wanting to use this with popovers- is that the case? If so, are you also using the popover polyfill as well? It may be that the simplest place to put the logic for implicit popover anchors is in the popover polyfill, and not here, since that is already tracking the connection between the button and the popover. 

That said, code contributions are welcome, and [financial support](https://opencollective.com/oddbird-open-source) (especially from companies that plan to use this in production) would also be helpful in adding this support.
